CODEOWNERS

```diff
@@ -0,0 +1,309 @@
+# Memcached integration
+#-----------------------------------------------------------------
+geode-memcached/**                                                @nabarunnag @DonalEvans
+
+#-----------------------------------------------------------------
+# Misc Utilities
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/util/concurrent/**        @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/shared/**                 @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/sequencelog/**            @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/concurrent/**             @nabarunnag @boglesby
+geode-core/**/org/apache/geode/distributed/internal/unsafe/**     @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/lang/**                   @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/monitoring/**             @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/exception/**              @nabarunnag @boglesby
+geode-core/**/org/apache/geode/lang/**                            @nabarunnag @boglesby
+geode-core/**/org/apache/geode/ra/**                              @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/util/**                   @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/cache/vmotion/**          @nabarunnag @boglesby
+geode-core/**/org/apache/geode/internal/jndi/**                   @nabarunnag @boglesby
+geode-common/**                                                   @nabarunnag @boglesby
+geode-unsafe/**                                                   @nabarunnag @boglesby
+
+#-----------------------------------------------------------------
+# Persistence
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/cache/persistence/**               @kirklund @dschneider-pivotal
+geode-core/**/org/apache/geode/internal/cache/persistence/**      @kirklund @dschneider-pivotal
+geode-core/**/org/apache/geode/internal/cache/backup/**           @kirklund @dschneider-pivotal @agingade
+
+#-----------------------------------------------------------------
+# Region Version Vectors - used for sychronization on 
+# member failures and persistent recovery
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/versions/**         @dschneider-pivotal @gesterzhou
+
+
+#-----------------------------------------------------------------
+# WAN messaging and queues
+#-----------------------------------------------------------------
+geode-wan/**                                                      @gesterzhou @jujoramos
+geode-core/**/org/apache/geode/cache/asyncqueue/**                @gesterzhou @jujoramos
+geode-core/**/org/apache/geode/cache/wan/**                       @gesterzhou @jujoramos
+geode-core/**/org/apache/geode/internal/cache/wan/**              @gesterzhou @jujoramos
```

Comment:
I can certainly try to help whenever/however I can with reviewing `PRs` but, truth be told, I'm not working that much on Geode code nowadays and wouldn't be fair to leave all the work to @gesterzhou alone.